### PR TITLE
Tp 1 test the application

### PR DESCRIPTION
**Pull request by lakviat**

### CODE REVIEW

File: src/main/java/apollo/TestApollo.java [hunk 1]
```diff
@@ -15,7 +15,6 @@
 
     @Override
     public void DigitCount() {
-
         //This is my code i want to merge with master branch, and this is what i done so far
```
Comment (arifsalamci) [inline src/main/java/apollo/TestApollo.java]: Please remove this one as well

Comment (ZhazgulOmurbekova) [inline src/main/java/apollo/TestApollo.java]: please remove this line

Comment (lakviat) [reply src/main/java/apollo/TestApollo.java]: Done

Comment (lakviat) [reply src/main/java/apollo/TestApollo.java]: Done please check last commit

**Review by ZhazgulOmurbekova — APPROVED** [review]

**Pull request merged**